src/components/Page/Page.tsx

```diff
@@ -23,6 +23,12 @@ export interface Props extends HeaderProps {
   fullWidth?: boolean;
   /** Decreases the maximum layout width. Intended for single-column layouts */
   singleColumn?: boolean;
+  /**
+   * Force render in page and do not delegate to the app bridge TitleBar action
```

Comment:
I'm thinking from the perspective of someone without any context on the issue, would this description be helpful for that person?

Reply:
I’m biased – what do you think? If someone, without context, uses this prop and the TitleBar no longer renders, I’d like them to know how they can still use TitleBar. Maybe we can add a link to [TitleBar](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/titlebar)?

Reply:
We could link to TitleBar, but that does feel a bit weird too. Perhaps we should add something to here: https://polaris.shopify.com/components/structure/page#section-use-in-an-embedded-application, and link to that?

Reply:
Follow the doc format like so and we can support these tags later in the style guide: https://github.com/Shopify/polaris-react/blob/ad4b8bdf06b1159cbccfba45125ef5bdf0f2f84b/src/components/AppProvider/types.ts#L24